sentinel-core/src/main/java/com/alibaba/csp/sentinel/slots/block/degrade/DegradeRule.java

```diff
@@ -104,11 +103,11 @@ public DegradeRule setCount(double count) {
         return this;
     }
 
-    public boolean isCut() {
-        return cut;
+    private boolean isCut() {
+        return cut.get();
     }
 
-    private void setCut(boolean cut) {
+    private void setCut(AtomicBoolean cut) {
```

Comment:
Please also keep the original signature like:

```suggestion
    private void setCut(boolean b) {
        this.cut.set(b);
    }
```